micromamba/src/umamba.cpp

```diff
@@ -63,6 +63,13 @@ set_umamba_command(CLI::App* com, mamba::Configuration& config)
     CLI::App* remove_subcom = com->add_subcommand("remove", "Remove packages from active environment");
     set_remove_command(remove_subcom, config);
 
+    // `uninstall` is an alias for `remove`
+    CLI::App* uninstall_subcom = com->add_subcommand(
+        "uninstall",
+        "Remove packages from active environment"
+    );
+    set_remove_command(uninstall_subcom, config);
```

Comment:
Note for reviewers: has CLI11 a feature for aliases or not?

Reply:
Yes, you can use `subcom->alias("uninstall");` instead of adding a new subcommand.